R/LinRegLA_adapt.R

```diff
@@ -0,0 +1,17 @@
+LinRegLA_adapt<- function(model, particles = 1000, resampTol = 0.5, tempTol = 0.9) {
+
+    if (model ==1){
+        Data <- cbind(radiata$y,radiata$x1)
+    } else if (model == 2){
+        Data <- cbind(radiata$y,radiata$x2)
+    } else{
+        stop("Please choose a valid model (1 or 2).")
+    }
+    
+    res <- LinRegLA_adapt_impl(as.matrix(Data), particles, resampTol, tempTol)
+
+    invisible(res)
+}
+
+## silence a NOTE from 'R CMD check --as-cran'
```

Comment:
Further to earlier discussion about this....

If I'm understanding a somewhat aged discussion correctly, BDR seems to suggest that we should just use the fully-qualified name to refer to the data set (http://r.789695.n4.nabble.com/no-visible-binding-for-global-variable-for-data-sets-in-a-package-tt4696053.html#a4696100) and use RcppSMC::radiata rather than radiata.

If I understood correctly, that obviates the need for a workaround like this. 

Although my instinct is to do as BDR directs unless there's a good reason to do otherwise, as with all such things I bow to @eddelbuettel and his vastly more extensive knowledge, so it's quite possible that I've misunderstood entirely or that there is some other reason for explicitly dealing with the checker in this way.